.github/workflows/issues.yml

```
@@ -0,0 +1,23 @@
+name: Workflows for opened and reopened github issues
+
+on:
+  issues:
+    types: [opened, reopened]
+jobs:
+  message:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: 'echo ${{ github.event.issue.title }}'
+    - name: Send a stream message
+      uses: zulip/github-actions-zulip/send-message@main
```

I would prefer to use a [fixed version](https://github.com/zulip/github-actions-zulip/releases/tag/v0.1.2) instead of latest `main` commit 
```suggestion
      uses: zulip/github-actions-zulip/send-message@v0.1.2
```

---

I would prefer too, but `v0.1.2` does not work, I believe the introduction of `send-message` is not yet packaged correctly by the zulip org.

---

actually done with 013ecb5